### DISCUSSION TITLE
additional layer for quickly assign

### DISCUSSION THREAD

**Discussion by opaldes** [nickcoutsos/keymap-editor#187] [Ideas]
Hi, 
first appretiate your work with the mapper.
I think there should be another layer where I can quickly assign the zmk specific keypresses, aka bluetooth profiles, reset and output behaviors trans none etc.. If I have time I will open up a PR with the changes but wanted to check if the change would be appretiated.

**Comment by nickcoutsos** [comment]
Hi

I'm not taking PRs, the public source is extremely outdated at this point and I'm just doing the development on my own.

Could you clarify what you mean by additional layer here?

**Reply by opaldes** [reply]
Currently you can only choose qwerty, colemak and dvorak as available options for the quick assign when adding layers, they are called Keymap palette I think.
It would be nice if there would be a "system" Keymap palette where the bluetooth buttons and other could be quick assigned aswell. Basicly all the buttons missing from the already available ones.

**Reply by nickcoutsos** [reply]
Ohhh I see what you mean. It's an interesting thought, I know other editors have that kind of approach. I think it's possible but it may take some doing since the current version specifically separates keycodes and behaviors. Let me give it some thought.
